layouts/css/page-modules/_header.styl

```diff
@@ -69,11 +69,16 @@ header
       float left
       padding 0
       margin 0
+    nav
+      margin 0 20px
     a,
     a:link,
     a:active
       padding 0
 
+  .nav-foundation
```

Comment:
Why not move L29 (with a copy of L28) to the "large" media breakpoint (around L64)
https://github.com/nodejs/nodejs.org/blob/0c985703cb0f36eee0ba09f357dab55c9241506c/layouts/css/page-modules/_header.styl#L28-L29

Reply:
Looking around the codebase, I had the feeling that it was mostly “desktop-first” which is why I opted for this variant. However, I agree that your suggestion makes more sense in this case. Changed 👍